app/piping/plumber.py

```diff
@@ -2,6 +2,8 @@
 
 import re
 
+from flask import render_template_string
```

Comment:
Maybe we could do this as a part of removing plumbing stuff. Spoke to @iwootten about this and it was making testing quite difficult as a part of the testing.